src/main/java/com/example/sixthmafiabot/exceptions/Abstract/CustomMapException.java

```diff
@@ -0,0 +1,18 @@
+package com.example.sixthmafiabot.exceptions.Abstract;
+
+import lombok.Getter;
+
+import java.util.Map;
+
+@Getter
+public abstract class CustomMapException extends BaseCustomException{
+
+    protected final Map <String, String> exceptionMap;
+
+    public CustomMapException(Map<String, String> exceptionMap) {
+
+        super(exceptionMap.toString());
```

Comment:
remove to string, we do not need string representation

Reply:
+